products/sle/main.pm

```diff
@@ -631,14 +636,17 @@ if (is_kernel_test()) {
         load_baremetal_tests();
     }
     load_kernel_tests();
+    baremetal_cleanup();
```

Comment:
I wonder if there is a better way to make sure cleanup is called after each tests, maybe in post_run/post_fail hooks in the base class? I don't like much we have to load this test every time... and the test as such only calls `$self->host_unlock();`

Reply:
I cannot imagine how a test module can know it is the last one and should cleanup?